superset/assets/src/components/FilterableTable/FilterableTableStyles.css

```diff
@@ -72,3 +72,10 @@
 }
 .even-row { background: #f2f2f2; }
 .odd-row { background: #ffffff; }
+.header-style {
+  direction: rtl;
+  overflow: hidden;
```

Comment:
Should we do `overflow: scroll` here instead? I know it doesn't look nice, but at least it allows people to different between columns that have long names and similar prefixes.

Reply:
I tried adding that style but it looked extremely bad :(